hazelcast/include/hazelcast/client/EntryEvent.h

```diff
@@ -44,12 +44,13 @@ namespace hazelcast {
                 ADDED =  1,
                 REMOVED = 1 << 1,
                 UPDATED = 1 << 2,
-                EVICTED = 1 << 3 ,
-                EVICT_ALL = 1 << 4 ,
-                CLEAR_ALL = 1 << 5 ,
-                MERGED = 1 << 6 ,
-                EXPIRED = 1 << 7,
+                EVICTED = 1 << 3,
+                EXPIRED = 1 << 4,
```

Comment:
These numbers seems to be changed when going from Java server 3.12.x to 4.0.x. Your change is based on 4.0.x but client did not switch to work with that server version yet (protocol changes will be done in later PRs), hence the tests (for e.g. for expired) normally would not pass (we may be missing the tests)

Reply:
Reverted back to 3.12.x compatible types.